packages/eas-cli/src/commands/submit.ts

```diff
@@ -170,6 +170,12 @@ export default class BuildSubmit extends Command {
     const { exp } = getConfig(projectDir, { skipSDKVersionRequirement: true });
     const projectId = await getProjectIdAsync(exp);
 
+    logEvent(Event.ACTION, {
+      action: `eas submit`,
+      project_id: projectId,
+      platform,
+    });
```

Comment:
maybe it would be better to write something generic that sends those event in `authorizedCommand` for every command

Reply:
yeah, i was thinking about this... I'll pull it into the init method then